chsdi/lib/validation/__init__.py

```diff
@@ -61,6 +61,7 @@ def srid(self):
 
     @srid.setter
     def srid(self, value):
+        # 2to3 would wrap the map() inside a list(), which is not necessary here.
```

Comment:
Useless comment, it should be remove. In very near future we don't care anymore in python2.

Reply:
all those comments are now removed